lib/msal-browser/src/network/XhrClient.ts

```diff
@@ -38,7 +38,7 @@ export class XhrClient implements INetworkModule {
             };
 
             if (method === "GET" || method === "POST") {
-                xhr.send(body);
+                reqBody ? xhr.send(reqBody) : xhr.send();
```

Comment:
Prefer if statement over ternary (since you aren't using the return value).

Reply:
Also, potentially sending a body with a GET is nonsensical, I would change this as such:

```js
if ((method === "POST"  && reqBody) {
   xhr.send(reqBody);
} else {
  xhr.send();
}
```

Reply:
Yeah that's way better. Will add now.